### ISSUE TITLE
Error parsing schema

### ISSUE THREAD

**Issue by brianyang** [amplience/dc-static-blog-nextjs#9]
using the provided schemas, encountering an error when creating content: 

#### The content schema could not be processed.

<img width="708" alt="Screen Shot 2019-12-09 at 11 03 05 AM" src="https://user-images.githubusercontent.com/81162/70451522-83062280-1a73-11ea-81b6-7faea1ad46a7.png">

## steps to reproduce 

follow the instructions in the readme, up until: 

```
# Creating A Blog-List & Slot
Once you have installed and registered all of Content schemas, the next step is to create a blog-list Content item and a Slot. A slot is like a placeholder/pointer to your blog-list, it is also the content entry point when the application runs.

How to create a blog-list content item for your blog:

Navigate to the "Production" section
Select the repo where you have registered "blog-list.json"
Click "Create content"
Select the "Blog List" (or whatever label to assigned to the "blog-list.json" content type)
```

## Expected 
feature works

## Actual
error prompt shown

**Comment by osecluna** [comment]
Hi @brianyang to clarify can you answer these:

-  which content type were you creating the content from?
-  which schema was it using?
-  was the schema created in the schema editor (internal) or used via a url (external)?
-

**Comment by brianyang** [comment]
hi @osecluna I'm following the readme and creating a blog-list content type.

i'm using this schema https://raw.githubusercontent.com/amplience/dc-static-blog-nextjs/master/schemas/blog-list.json

the schema was used via a url

**Comment by jhabdas** [comment]
Off-topic: @brianyang you do not have an email address listed on your website so I had to sign-up for this social service to contact you. I believe I've contacted you in the past but I want to reiterate, you [are](http://brianyang.com/awesome-react-components/) illegally using [my content](https://habd.as/awesome-react-components/). Please remove my post or abide by the license indicated at the bottom of the page you copied. And bro, please add an email address to your website so people don't have to jump through hoops to contact you.

**Comment by easen-amp** [comment]
Hi @brianyang,

Just to clarify you created the Content types using the github hosted URL, is this correct? You can check this by navigating to each Content type and looking at the Content type schema field, internal means the schema is hosted within your Dynamic Content Hub, external means it's hosted external (Github for example).

The README describes creating internally hosted schemas (Content type schemas). However that being said it should still work using externally hosted schemas (i.e "external" content type schemas). 

Can you check to ensure you all 7 Content types registered?

**Comment by easen-amp** [comment]
Hi @brianyang,

We have managed to replicate the issue following the documentation, I think the problem is that the docs missed out 2 steps during the creation of the content type schemas.

I've created PR #10, please can follow the updated docs in the PR and let me know if this resolves your issue.

Thanks.

**Comment by easen-amp** [comment]
One thing to point out, as you've created your Content types you will have to sync them with the updated schemas, see https://docs.amplience.net/integration/refreshingcontenttypes.html?h=sync for information on how to do this.

**Comment by brianyang** [comment]
hi @easen-amp,

Thanks for the clarification. 

The README says to create the schema and enter the schema ID. the schema ID for this demo lists the github url. how do you create an internally hosted schema? when you click on the 'create schema' button it opens a modal that asks you to enter a schema ID, a unique URI. 

it appears that all 7 Content types are registered. When i attempt to register on of these content types it says: "Duplicate item" 

it appears these content types are registered but when i try to create content with one of these, it says: "Error parsing schema"

**Comment by brianyang** [comment]
issue resolved with PR #10
